bookwyrm/views/user.py

```diff
@@ -142,6 +142,49 @@ def get(self, request, username):
         return TemplateResponse(request, "user/relationships/following.html", data)
 
 
+class Statistics(View):
+    """get statistics about user"""
+
+    def get(self, request, username):
+        """show statistics to oneself"""
+        # TODO: How to handle ActivitypubResponse here?
```

Comment:
Since this view doesn't need to federate, you won't need to handle that

Reply:
So, no ActivityPub request will ever hit this code path?
I haven't figured out how that plays along, yet. Are there hints in the `urls.py`?

Reply:
Hm, I'm not sure what is the best way to know now that you mention it. Any view that represents a model instance that uses the `ActivitypubMixin` generally is the only view that deals with `ActivityPub` -- so most views don't.